### PR TITLE
Fix lint issues

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@nalind @runcom PTAL

File: test/copyimg/copyimg.go [hunk 1]
```diff
@@ -133,7 +135,10 @@ func main() {
 			logrus.Errorf("error loading signature policy: %v", err)
 			os.Exit(1)
 		}
-		defer policyContext.Destroy()
+		defer func() {
+			_ = policyContext.Destroy()
+
```
Comment (runcom) [inline test/copyimg/copyimg.go]: nit, blank line

File: test/copyimg/copyimg.go [hunk 2]
```diff
@@ -110,7 +110,9 @@ func main() {
 				logrus.Errorf("error opening storage: %v", err)
 				os.Exit(1)
 			}
-			defer store.Shutdown(false)
+			defer func() {
+				_, _ = store.Shutdown(false)
```
Comment (runcom) [inline test/copyimg/copyimg.go]: silly lint is complaning about this :confused: does it still complain if you just write `store.Shutdown(false)` here? maybe it complaints just after a `defer`

File: server/sandbox_run.go [hunk 3]
```diff
@@ -408,10 +408,7 @@ func (s *Server) setPodSandboxMountLabel(id, mountLabel string) error {
 	}
 	storageMetadata.SetMountLabel(mountLabel)
 	err = s.storage.SetContainerMetadata(id, storageMetadata)
-	if err != nil {
-		return err
-	}
-	return nil
+	return err
```
Comment (runcom) [inline server/sandbox_run.go]: `return s.storage.SetContainerMetadata(id, storageMetadata)` directly

File: test/copyimg/copyimg.go (hunk 1 above)
Comment (mrunalp) [inline test/copyimg/copyimg.go]: Fixed.

File: server/sandbox_run.go (hunk 3 above)
Comment (mrunalp) [inline server/sandbox_run.go]: done.

**Comment by runcom** [comment]
LGTM if Travis's happy :smile:

File: test/copyimg/copyimg.go (hunk 2 above)
Comment (mrunalp) [inline test/copyimg/copyimg.go]: yeah, it fails :/

```
test/copyimg/copyimg.go:114:19:warning: error return value not checked (store.Shutdown(false)) (errcheck)
```

**Comment by sameo** [comment]
Lint is picky sometimes, yes...
LGTM

**Comment by mrunalp** [comment]
Tests are green. Merging.

**Pull request merged**